src/test/ui/borrowck/borrowck-move-out-of-overloaded-auto-deref.mir.stderr

```diff
@@ -1,8 +1,8 @@
-error[E0507]: cannot move out of borrowed content
+error[E0507]: cannot move out of an `Rc`
   --> $DIR/borrowck-move-out-of-overloaded-auto-deref.rs:17:14
    |
 LL |     let _x = Rc::new(vec![1, 2]).into_iter();
-   |              ^^^^^^^^^^^^^^^^^^^ cannot move out of borrowed content
+   |              ^^^^^^^^^^^^^^^^^^^ cannot move out of an `Rc`
```

Comment:
so..much...nicer...

Reply:
I wonder, though: if we stay specialized, then we can offer a hint like "you might want to consider `Rc::try_unwrap`".

Reply:
I've added a `try_unwrap` suggestion - are you happy with the wording?